apps/charterafrica/src/components/Postcard/PostCard.js

```diff
@@ -0,0 +1,62 @@
+import { Card, CardContent, CardMedia } from "@mui/material";
+import PropTypes from "prop-types";
+import { forwardRef } from "react";
+
+import LineClampedRichTypography from "../LineClampedRichTypography";
+
+const PostCard = forwardRef((props, ref) => {
+  const { author, date, image, title, sx } = props;
+
+  return (
+    <Card elevation={1} sx={sx} ref={ref}>
+      <CardMedia
+        image={image.url}
+        title={image.filename}
+        sx={{ height: 200 }}
+      />
+      <CardContent>
+        <LineClampedRichTypography
+          color="neutral.dark"
+          typograpy={{ md: "h5SemiBold" }}
+          variant="h5SmallSemiBold"
+          textAlign="left"
+          sx={{ mb: 2.5 }}
+          html={false}
+          lineClamp={3}
```

Comment:
You need to set title height, otherwise each card may end up with different height i.e. misaligned (Spotlight handles this).
![Screenshot from 2023-02-03 11-18-10](https://user-images.githubusercontent.com/1779590/216549080-bf906b5a-fd84-4479-9cf4-912a93963306.png)